File: majority-element.js
var majorityElement = function(nums) {
    
    let n = nums.length
    let obj = {}
    
    for (let i = 0; i < nums.length; i++) {
    if (obj[nums[i]]) {
        obj[nums[i]]++
    } else {
        obj[nums[i]] = 1
    }
    }
    for (let el in obj) {
        if (obj[el] > n/2) return el
    }
    };

File: Leetcode_Python/majority_element.py
class Solution(object):
    def majorityElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        majority = float(len(nums))/2
        dictionary = {}
        for i in nums:
            if i in dictionary:
                dictionary[i] += 1
            else:
                dictionary[i] = 1

        for i in dictionary:
            if (dictionary[i] > majority):
                return i